app/commands/process_raw_event.rb

```diff
@@ -0,0 +1,129 @@
+class ProcessRawEvent
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def initialize(raw_event_id)
+    @raw_event_id = raw_event_id
+  end
+
+  def call
+    raw_event = RawEvent.find_by(id: raw_event_id)
+    process_raw_event(raw_event) if raw_event.present?
+  end
+
+  def process_raw_event(raw_event)
+    application = raw_event.application
+
+    parsed_data = parse_raw_data(raw_event)
+
+    meta = parsed_data[:meta]
+    details = parsed_data[:details]
+    details = add_transactions_to_details(details)
+    scenario_hash = generate_scenario_hash(details)
+
+    event_source = find_or_create_event_source(application: application, meta: meta)
+    scenario = find_or_create_scnario(event_source: event_source, hash: scenario_hash)
+    create_request(scenario: scenario, raw_event: raw_event, meta: meta)
+
+    events = details.each_with_index.map do |row, index|
+      event_hash(scenario: scenario, details: row, index: index)
+    end
+
+    store_events(events)
+  end
+
+  def store_events(events)
```

Comment:
Unused method argument - `events`. If it's necessary, use `_` or `_events` as an argument name to indicate that it won't be used. You can also write as `store_events(*)` if you want the method to accept any arguments but don't care about them.